project.clj

```diff
@@ -1,4 +1,4 @@
-(def ps-version "5.3.0")
+(def ps-version "5.3.0-SNAPSHOT")
```

Comment:
I think this should point to the next release, ie `5.3.1-SNAPSHOT`

Reply:
Oh yeah. Updated.